src/Microsoft.DotNet.Interactive.Tests/KernelExtensions.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.DotNet.Interactive.Tests
+{
+    public static class KernelExtensions
+    {
+        public static IKernel FindKernel(this IKernel kernel, Language name)
```

Comment:
There's a public method in `Microsoft.DotNet.Interactive` to do this.

Reply:
Oh, I see, this is just shimming the enum. 

You can dedupe the string conversion logic by calling `name.LanguageName()`.

Reply:
perfect